internal/exec_unix.go

```diff
@@ -56,3 +56,11 @@ func WaitTimeout(c *exec.Cmd, timeout time.Duration) error {
 	// Otherwise there was an error unrelated to termination.
 	return err
 }
+
+func GracefulStop(cmd *exec.Cmd, timeout time.Duration) {
```

Comment:
Move these to execd plugin, also suggest calling this something like `SignalStop`.

Reply:
Moved it, but I kind of like the name; signalStop sounds like ... stop sending or listening for signals or something.